content/200-concepts/300-more/050-environment-variables/100-managing-env-files-and-setting-variables.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: 'Managing .env files and setting variables'
+metaTitle: 'Managing .env files and setting variables'
+metaDescription: 'Learn how to manage .env files and set environment variables'
+tocDepth: 2
+---
+
+<TopBlock>
+
+Although [Prisma creates an `.env` file for you upon installation](./#how-does-prisma-use-environment-variables), there can be times when you want to:
+
+- Bypass `.env` files and let Prisma use the system environment variables directly
+- Use `.env` files from a location that the Prisma CLI does not check [by default](./#using-env-files)
+- [Use multiple `.env` file](./using-multiple-env-files)
+
+</TopBlock>
+
+## Using the system environment directly
+
+Because Prisma reads from the system's environment when looking for environment variables, it's possible to skip using `.env` completely and create them manually on your local system. 
+
+<Admonition type="info">
+
+The following examples will use setting the `DATABASE_URL` environment variable which is often used for the database connection URL.
+
+</Admonition>
+
+### Manually set an environment variable on a Unix system
```

Comment:
Do MacOS users self identify as using a Unix system?

Reply:
Im not sure 😄 But macos is based on unix. I have changed the title to `Manually set an environment variable on a Mac/Linux system`